.github/workflows/scripts/verify-php-version.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+#
+# Verify that the PHP version in the launched WP ENV environment is equal to expected.
```

Comment:
I added this bash script to verify if the workflow was indeed able to set up the correct PHP version in the `wp-env` localhost site. I've encountered some problems working with our WP ENV localhost setup like Jon and Jamel did in the past (like WP ENV not allowing us to run certain wp cli commands), and I think it's good for us to have a way to confirm if the localhost site was set up correctly, just like the PHP version.